diff -up mupdf-1.14.0-source/source/fitz/color-lcms.c.omv~ mupdf-1.14.0-source/source/fitz/color-lcms.c
--- mupdf-1.14.0-source/source/fitz/color-lcms.c.omv~	2019-03-02 17:49:12.980274182 +0100
+++ mupdf-1.14.0-source/source/fitz/color-lcms.c	2019-03-02 17:51:08.616931618 +0100
@@ -240,7 +240,7 @@ fz_lcms_init_link(fz_cmm_instance *insta
 
 	if (prf == NULL)
 	{
-		link->cmm_handle = cmsCreateTransformTHR(cmm_ctx, src->cmm_handle, src_data_type, dst->cmm_handle, des_data_type, rend->ri, flag);
+		link->cmm_handle = cmsCreateTransform(cmm_ctx, src->cmm_handle, src_data_type, dst->cmm_handle, des_data_type, rend->ri, flag);
 		if (!link->cmm_handle)
 			fz_throw(ctx, FZ_ERROR_GENERIC, "cmsCreateTransform failed");
 	}
@@ -251,13 +251,13 @@ fz_lcms_init_link(fz_cmm_instance *insta
 		 */
 		if (src == prf)
 		{
-			link->cmm_handle = cmsCreateTransformTHR(cmm_ctx, src->cmm_handle, src_data_type, dst->cmm_handle, des_data_type, INTENT_RELATIVE_COLORIMETRIC, flag);
+			link->cmm_handle = cmsCreateTransform(cmm_ctx, src->cmm_handle, src_data_type, dst->cmm_handle, des_data_type, INTENT_RELATIVE_COLORIMETRIC, flag);
 			if (!link->cmm_handle)
 				fz_throw(ctx, FZ_ERROR_GENERIC, "cmsCreateTransform failed");
 		}
 		else if (prf == dst)
 		{
-			link->cmm_handle = cmsCreateTransformTHR(cmm_ctx, src->cmm_handle, src_data_type, prf->cmm_handle, des_data_type, rend->ri, flag);
+			link->cmm_handle = cmsCreateTransform(cmm_ctx, src->cmm_handle, src_data_type, prf->cmm_handle, des_data_type, rend->ri, flag);
 			if (!link->cmm_handle)
 				fz_throw(ctx, FZ_ERROR_GENERIC, "cmsCreateTransform failed");
 		}
@@ -277,7 +277,7 @@ fz_lcms_init_link(fz_cmm_instance *insta
 				lcms_prf_cs = 0;
 			prf_num_chan = cmsChannelsOf(cmm_ctx, prf_cs);
 			prf_data_type = (COLORSPACE_SH(lcms_prf_cs) | CHANNELS_SH(prf_num_chan) | BYTES_SH(num_bytes));
-			src_to_prf_link = cmsCreateTransformTHR(cmm_ctx, src->cmm_handle, src_data_type, prf->cmm_handle, prf_data_type, rend->ri, flag);
+			src_to_prf_link = cmsCreateTransform(cmm_ctx, src->cmm_handle, src_data_type, prf->cmm_handle, prf_data_type, rend->ri, flag);
 			if (!src_to_prf_link)
 				fz_throw(ctx, FZ_ERROR_GENERIC, "cmsCreateTransform failed");
 			src_to_prf_profile = cmsTransform2DeviceLink(cmm_ctx, src_to_prf_link, 3.4, flag);
@@ -288,7 +288,7 @@ fz_lcms_init_link(fz_cmm_instance *insta
 			hProfiles[0] = src_to_prf_profile;
 			hProfiles[1] = prf->cmm_handle;
 			hProfiles[2] = dst->cmm_handle;
-			link->cmm_handle = cmsCreateMultiprofileTransformTHR(cmm_ctx, hProfiles, 3, src_data_type, des_data_type, INTENT_RELATIVE_COLORIMETRIC, flag);
+			link->cmm_handle = cmsCreateMultiprofileTransform(cmm_ctx, hProfiles, 3, src_data_type, des_data_type, INTENT_RELATIVE_COLORIMETRIC, flag);
 			cmsCloseProfile(cmm_ctx, src_to_prf_profile);
 			if (!link->cmm_handle)
 				fz_throw(ctx, FZ_ERROR_GENERIC, "cmsCreateMultiprofileTransform failed");
@@ -317,7 +317,7 @@ fz_lcms_new_instance(fz_context *ctx)
 	DEBUG_LCMS_MEM(("Context Creation:: mupdf ctx = %p lcms ctx = %p \n", (void*) ctx, (void*) cmm_ctx));
 	if (cmm_ctx == NULL)
 		fz_throw(ctx, FZ_ERROR_GENERIC, "cmsCreateContext failed");
-	cmsSetLogErrorHandlerTHR(cmm_ctx, fz_lcms_log_error);
+	cmsSetLogErrorHandler(cmm_ctx, fz_lcms_log_error);
 	return (fz_cmm_instance *)cmm_ctx;
 }
 
@@ -341,7 +341,7 @@ fz_lcms_init_profile(fz_cmm_instance *in
 	DEBUG_LCMS_MEM(("@@@@@@@ Create Profile Start:: mupdf ctx = %p lcms ctx = %p \n", (void*)ctx, (void*)cmm_ctx));
 
 	size = fz_buffer_storage(ctx, profile->buffer, &data);
-	profile->cmm_handle = cmsOpenProfileFromMemTHR(cmm_ctx, data, (cmsUInt32Number)size);
+	profile->cmm_handle = cmsOpenProfileFromMem(cmm_ctx, data, (cmsUInt32Number)size);
 	if (profile->cmm_handle == NULL)
 	{
 		profile->num_devcomp = 0;
